charts/shoot-cloud-config/charts/downloader/templates/cloud-config/_download-cloud-config.sh

```diff
@@ -8,8 +8,6 @@ DIR_CLOUDCONFIG="/var/run/coreos"
 DIR_KUBELET="/var/lib/kubelet"
 
 PATH_KUBECONFIG="$DIR_CLOUDCONFIG_DOWNLOADER/kubeconfig"
-PATH_CLOUDCONFIG="$DIR_CLOUDCONFIG/cloud_config.yml"
-PATH_CLOUDCONFIG_OLD="$DIR_CLOUDCONFIG/cloud_config.old.yml"
 
 mkdir -p "$DIR_CLOUDCONFIG" "$DIR_KUBELET"
```

Comment:
The downloaded script can do it on its own, right?
Can we remove this (same applies for the used variables)?